jenkins-pipelines/manager-sanity.jenkinsfile

```diff
@@ -7,6 +7,7 @@ longevityPipeline(
     params: params,
 
     backend: 'aws',
+    scylla_version: '3.1.2'
```

Comment:
this won't work, `vars/longevityPipeline.groovy` doesn't support reading `scylla_version` from this file